app/src/main/java/com/veniosg/dir/android/adapter/FileListViewHolder.java

```diff
@@ -41,7 +41,7 @@ void bind(String filePath, OnItemClickListener listener) {
 
         primaryInfo.setText(item.getName());
         secondaryInfo.setText(item.getFormattedModificationDate(context));
-        tertiaryInfo.setText(isDirectory ? "" : item.getFormattedSize(context, false));
+        tertiaryInfo.setText(isDirectory ? item.getNumberOfItems(context) : item.getFormattedSize(context, false));
```

Comment:
Used here too. Ideally `getNumberOfItems` and `getFormattedSize` will now be private.

Reply:
Done!